README.platform

```diff
@@ -85,6 +85,24 @@ There is no official registry of 3rd party event numbers, so if this
 number is already in use on your system, you may change it at build time
 by configure'ing --with-cflags=-DAUE_openssh=32801 then rebuilding.
 
+HPE NonStop
+-----------
+OpenSSH is primarily useful on the NonStop platform primarily as a client.
+There is already an implementation with the operating system supporting most
+sshd functions. To build on the platform, standard configure rules apply with
+the follwing requirements:
+
+_RLD_LIB_PATH, includes, and --withssl-dir needs to specify the path to
+OpenSSL on the platform. c99 and c11 are recommended. The CPPFLAGS variable
+should include -Wextensions and -DNSIG=32. --disable-utmpx is required. Once
+configured, the following changes are required in config.h because the configure
+script needs help:
```

Comment:
what does ./config.guess report on this platform?

You can add that to the giant 'case "$host" in' block (alphabetical order please) and start putting your platform-specific stuff in it.

Reply:
`nse-tandem-nskJ06` and `nse-tandem-nskL21` depending on the variant. There are other variations as well.

Reply:
That would be good for the required items. I will gather those up and supply what `config.log` shows (as it fails).

Reply:
sounds like a "nse-tandem-*) case would be a good starting point.  It's not going to conflict with anything else and you can always make it more specific if necessary.

Reply:
Actually `nsx-tandem-nskL21`. So "ns[ex]-tandem-nsk*" Or even, to cover older and meanwhile outdated systems "ns[rex]-tandem-nsk*"

Reply:
All that is in newer versions of these configure tools. (and since a few years now)

Reply:
Yet we find this issue time and time again.

Reply:
I don't think we need `nsr` anymore. Those systems are way off support, but I'll keep them in case someone wants to play on a dusty system. I'm not going to maintain `nsr` builds. Nope. :)

Reply:
We need to keep some of the knobs, like `--with-ssl-dir` because we don't know which version one will be used. 1.0.2, 1.1.1, and 3.0.x are all active (although I don't think OpenSSH is on 3.0.x yet).

Reply:
The nsr part most probably is in config.guess and config.sub already, no need to remove it ;-)

Reply:
This file should now be up to speed.